TWLight/resources/templates/resources/partner_detail.html

```diff
@@ -196,6 +196,32 @@ <h1>{{ object }}</h1>
         </div>
       </div>      
     </li>
+    {% if object.additional_resources %}    
+      <li class="timeline-inverted">
+        <div class="timeline-badge"><i class="fa fa-file"></i>
+        </div>
+        <div class="timeline-panel">
+          <div class="timeline-body">
+            <p>
+            {% comment %}
```

Comment:
This isn't really anything for you to change, but we should look into this and see if this comment is still accurate, or if the issue has been fixed, given that it's over a year old.

Reply:
I did some digging and looks like the issue is [fixed](https://code.djangoproject.com/ticket/27882). Although, I did try using None with no luck. Update Django maybe?

Reply:
Should be available in [version 2.0.x] (https://github.com/django/django/commit/7a7b331cd5975477597dac4dec7ee0ddb67f59e0), I think

Reply:
Ah, we're not going to be moving to 2.0 for a while yet, thanks for checking!